index.js

```diff
@@ -4275,7 +4229,9 @@
   //. Just ([1, 2, 3])
   //. ```
   function parseJson(pred) {
-    return B (filter (pred)) (B (eitherToMaybe) (encase (JSON.parse)));
+    return B (filter (pred))
+             (B (either (K (Nothing)) (Just))
+                (encase (JSON.parse)));
```

Comment:
I find the new implementation just as clear as the existing one. `either (K (Nothing)) (Just)` *is* the natural transformation from either to maybe. :)